docs/docs/design/grid-tracker.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: Grid Tracker
+permalink: /design/grid-tracker/
+---
+
+# Statement Distribution: Grid Tracker
+
+Grid tracker is a knowledge data struct that stores authorities for a given relay parent.
+
+- **received**: maps a validator index to a set of candidate hashes where each received candidate hash has a summary (like a gist) of the candidate, that is called `Manifest Summary`, in this summary we have the parent hash, group index and a statement filter.
+- **confirmed backed**: maps a candidate hash, that we already have marked as backed, to a set of informations that helps us to understand the group where this candidate was originated as well as how statements already was circulated and what statements for that candidate are pending to what validator (possible recipients) and what could be the possible senders, that is called `MutualKnowledge`
+- **unconfirmed**: maps candidates not backed to a list of sender (validator index) and group index
+- **pending manifests**: stores a list of manifests that should be sent to a specific validator.
```

Comment:
Can you elaborate please on what manifest is?

Reply:
Sure! Added a description on what a manifest is